package.json

```diff
@@ -53,6 +53,7 @@
     "npm-run-all": "^4.1.5",
     "postcss": "^8.4.5",
     "rimraf": "^3.0.2",
+    "rollup-plugin-copy": "^3.4.0",
```

Comment:
with this we dont need to declare `script.global.js` ? is that the whole point ?